### PR TITLE
Update Homepage

### PR DESCRIPTION
**Pull request by garaynakca**
We are updating the homepage with card-based calls to action.  

----
:books: Documentation preview :books:: https://civicactions-handbook--1040.org.readthedocs.build/en/1040/

### CODE REVIEW

**Review by ContentBrewmaster — APPROVED** [review]

**Comment by dmundra** [comment]
@garaynakca and @ContentBrewmaster I think the header level should be fixed to follow the correct order if we want to merge this pull request.

**Comment by dmundra** [comment]
@garaynakca are you continuing to work on this branch? Then I recommend we convert it to a draft so we don't inadvertently merge it. What do you think?

**Comment by garaynakca** [comment]
> 

@dmundra Yes... still working on this one... hoping to add the mkdocs grid cards with our content.  Sorry I should have made a draft (learning)... thank you!

**Comment by dmundra** [comment]
Thank you @garaynakca. No sorries needed.

FYI, you can change it to a draft by clicking 'Convert to draft' below 'Reviewers' on the right sidebar.

File: README.md [hunk 1]
```diff
@@ -6,7 +6,47 @@ hide:
 
 # CivicActions Guidebook
 
-[![Documentation Status](https://readthedocs.org/projects/civicactions-handbook/badge/?version=latest)](https://guidebook.civicactions.com/en/latest/?badge=latest)
+<!--[![Documentation Status](https://readthedocs.org/projects/civicactions-handbook/badge/?version=latest)](https://guidebook.civicactions.com/en/latest/?badge=latest) -->
```
Comment (dmundra) [inline README.md]: @garaynakca if @grugnog agrees we can remove this or move it to a different page.

Comment (grugnog) [reply README.md]: I think we could just remove this, honestly - showing green badges/coverage etc is nice for an open source project that wants to encourage usage, but I am not sure it is that relevant to a documentation project like this.

Comment (garaynakca) [reply README.md]: Screen Shot 2023-02-24 at 11 12 13 AM
@dmundra I think @ContentBrewmaster wants to remove this "beta" in the header.  Can you point me in the right direction to do that.  Thanks!

Comment (dmundra) [reply README.md]: Ya @garaynakca. That comes from https://github.com/CivicActions/guidebook/blob/master/.config/mkdocs.yml#L1

Comment (ContentBrewmaster) [reply README.md]: @garaynakca any blockers for merging the new design that I can help with?

Comment (dmundra) [reply README.md]: I went ahead and removed 'Documentation Status'.

**Review by dmundra — APPROVED** [review]
Thanks @garaynakca for your work on the homepage!

**Pull request merged**